### ISSUE TITLE
Customization as described in the docs has no effect on harbor's look

### ISSUE THREAD

**Issue by rbicker** [goharbor/harbor#8273]
Hi

Following the "Customize the look & feel of Harbor" - guide (https://github.com/goharbor/harbor/blob/master/docs/customize_look%26feel_guide.md), I have customized and built harbor.

(On a side note: The json shown in the documentation is outdated, the properties were renamed as it seems).

**Expected behavior and actual behavior:**
I expected the look to change according to my adjustments. This did not happen, the website was created with the default looks (including the default logo instead of my custom one).

**Steps to reproduce the problem:**
After cloning the master branch, I have placed an image (custom-logo.svg) in src/portal/src/images and I adjusted src/portal/src/setting.json:
``` json
{
  "headerBgColor": "#205081",
  "headerLogo": "custom-logo.svg",
  "loginBgImg": "",
  "appTitle": "My Company Registry",
  "project": {
    "companyName": "my company",
    "projectName": "registry",
    "introduction": {
      "en-us": ""
    }
  }
}
```
I then built harbor using:
``` bash
make install GOBUILDIMAGE=golang:1.12.5 DEVFLAG=false COMPILETAG=compile_golangimage CLAIRFLAG=true NOTARYFLAG=true CHARTFLAG=true VERSIONTAG=mycompany-1.8.1
```

My customization seems to be included in the container but it does not get applied:
``` bash
[root@registry:/usr/local/src/harbor]# docker exec -it harbor-portal ls /usr/share/nginx/html/images
custom-logo.svg  helm-gray.svg  harbor-logo.svg
[root@registry:/usr/local/src/harbor]# 
[root@registry:/usr/local/src/harbor]# 
[root@registry:/usr/local/src/harbor]# docker exec -it harbor-portal cat /usr/share/nginx/html/setting.json
{
  "headerBgColor": "#205081",
  "headerLogo": "custom-logo.svg",
  "loginBgImg": "",
  "appTitle": "My Company Registry",
  "project": {
    "companyName": "my company",
    "projectName": "registry",
    "introduction": {
      "en-us": ""
    }
  }
}
```

**Versions:**
Please specify the versions of following systems.
- harbor version: 1.8.1 / git master branch
- docker engine version: 17.12.0-ce
- docker-compose version: 1.24.0

If you need any further investigations to help resolve the issue I am happy to help.

Thanks a lot!

**Comment by reasonerjt** [comment]
I guess the doc maybe no longer works.

**Comment by zhoumeina** [comment]
yes, it is no longer works. @jwangyangls , could you please take a look the code and make sure it is working?

**Comment by jwangyangls** [comment]
it has fixed in harbor 1.9.0
